runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -692,7 +692,44 @@ TR_J9ServerVM::getStaticReferenceFieldAtAddress(uintptrj_t fieldAddress)
 bool
 TR_J9ServerVM::stackWalkerMaySkipFrames(TR_OpaqueMethodBlock *method, TR_OpaqueClassBlock *clazz)
    {
+   if(!method)
+      {
+      return false;
+      }
+   
    JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
+
+   if (_compInfoPT->getClientData()->getOrCacheVMInfo(stream)->_jlrMethodInvoke == NULL || 
+         _compInfoPT->getClientData()->getOrCacheVMInfo(stream)->_jlrMethodInvoke == ((J9Method *) method))
+      {
+      return true;
+      }
+
+   if(!clazz)
+      {
+      return false;
+      }
+
+   TR_OpaqueClassBlock * jlrAccessibleObjectClass = _compInfoPT->getClientData()->getOrCacheVMInfo(stream)->_jlrAccessibleObjectClass;
+   if (jlrAccessibleObjectClass != NULL && TR_J9ServerVM::isInstanceOf( clazz, jlrAccessibleObjectClass ,false))
```

Comment:
I don't see `_jlrAccessibleObjectClass` set anywhere in the VM. We need to ask about this and maybe we can remove it completely

Reply:
I believe it's obsolete, in 2011 there was a commit "never initialize vm->jlrAccessibleObject again, and stop checking against it in stackwalks as we will always be J2SE_SHAPE_SUN"